common/templates/general.go

```diff
@@ -131,6 +131,14 @@ func KindOf(input interface{}, flag ...bool) (string, error) { //flag used only
 	}
 }
 
+func HasPrefix(input, prefix string) bool {
+	return strings.HasPrefix(input, prefix)
+}
+
+func HasSuffix(input, prefix string) bool {
+	return strings.HasSuffix(input, prefix)
+}
+
```

Comment:
```suggestion
```

Reply:
sounds like it could work, i'll test it tomorrow :ye: